src/vivarium_gates_nutrition_optimization_child/utilities.py

```diff
@@ -179,6 +179,33 @@ def get_norm_from_quantiles(
     return stats.norm(loc=mean, scale=sd)
 
 
+def get_lognorm_from_quantiles(
+    median: float, lower: float, upper: float, quantiles: Tuple[float, float] = (0.025, 0.975)
+) -> stats.lognorm:
+    """Returns a frozen lognormal distribution with the specified median, such that
+    (lower, upper) are approximately equal to the quantiles with ranks
+    (quantile_ranks[0], quantile_ranks[1]).
+    """
+    # Let Y ~ norm(mu, sigma^2) and X = exp(Y), where mu = log(median)
+    # so X ~ lognorm(s=sigma, scale=exp(mu)) in scipy's notation.
+    # We will determine sigma from the two specified quantiles lower and upper.
+
+    # mean (and median) of the normal random variable Y = log(X)
+    mu = np.log(median)
```

Comment:
I'm going to go ahead and assume this is all correct lol

Reply:
So did I because I just copy pasted it :)